DasharoModulePkg.dec

```diff
@@ -44,7 +44,8 @@
   gDasharoSystemFeaturesTokenSpaceGuid.PcdShowUsbMenu|FALSE|BOOLEAN|0x00000004
   gDasharoSystemFeaturesTokenSpaceGuid.PcdShowNetworkMenu|FALSE|BOOLEAN|0x00000005
   gDasharoSystemFeaturesTokenSpaceGuid.PcdShowChipsetMenu|FALSE|BOOLEAN|0x00000006
-  gDasharoSystemFeaturesTokenSpaceGuid.PcdDefaultNetworkBootEnable|FALSE|BOOLEAN|0x00000007
+  gDasharoSystemFeaturesTokenSpaceGuid.PcdShowPowerMenu|FALSE|BOOLEAN|0x00000007

```

Comment:
Now that I think of it we may have the power menu for desktop as well. We will need a separate PCD for laptop options like fans, battery charging, etc... Maybe we could add another PCD for laptop options or detect system type from SMBIOS (if it is laptop) and the show additional options. I would like to get it done now rather than break something later.

Reply:
Makes sense, added another pcd